spotipy/client.py

```diff
@@ -1453,8 +1453,25 @@ def new_releases(self, country=None, limit=20, offset=0):
             "browse/new-releases", country=country, limit=limit, offset=offset
         )
 
+    def category(self, category_id, country=None, locale=None):
+        """ Get info about a category
+
+            Parameters:
+                - category_id - The Spotify category ID for the category.
+
+                - country - An ISO 3166-1 alpha-2 country code.
+                - locale - The desired language, consisting of an ISO 639
+                  language code and an ISO 3166-1 alpha-2 country code, joined
+                  by an underscore.
+        """
+        return self._get(
+            "browse/categories" + category_id,
```

Comment:
does it need a slash between these two bits?

Reply:
Yes!